### ISSUE TITLE
Remove #[allow(unused)] from items in parachains runtime

### ISSUE THREAD

**Issue by rphmeier** [paritytech/polkadot#1211]
As some modules are being written before others, I have been tagging items as #[allow(unused)] to pass CI. Once all the modules are written, the #[allow(unused)] attributes should be removed.

**Comment by rphmeier** [comment]
This will depend on #1209 for the Sudo wrapper, or integration with the `Registrar`. Whichever comes first.

**Comment by rphmeier** [comment]
(This probably doesn't need its own PR and should be rolled into another one)

**Comment by rphmeier** [comment]
Superseded by #1171
